src/app/feed/feed-create-post/feed-create-post.component.ts

```diff
@@ -71,6 +72,25 @@ export class FeedCreatePostComponent implements OnInit {
     this._setRandomMovieQuote();
   }
 
+  emojiClick(event: any): void {
+    const trigger = event.currentTarget;
+    const picker = new EmojiButton({
+      theme: window.localStorage.getItem("theme") == "light" ? "light" : "dark",
```

Comment:
we should also set `rootElement` to some element in feed-create-post. for example, I set the id on the first div in `feed-create-post.component.html` and got that element by id and used that as the root element.  By doing this, we can pass all the theme variables to the emoji picker. 

Do this and then add styling to `.emoji-picker` to set background color, font colors, and borders.

Reply:
rootElement described here: https://emoji-button.js.org/docs/api/

Reply:
> we should also set `rootElement` to some element in feed-create-post. for example, I set the id on the first div in `feed-create-post.component.html` and got that element by id and used that as the root element. By doing this, we can pass all the theme variables to the emoji picker.
> 
> Do this and then add styling to `.emoji-picker` to set background color, font colors, and borders.

I added rootElement in, I am not sure how you are getting the element ID to set as root, so I will let you do that, or we can do in future to extend the theming.